### PR TITLE
Lap4 se 129

### PR DESCRIPTION
**Pull request by Bita-Fotovvat**
Linked the project card modal to the right place (marketplace task detail page).
Updated the style of project card modal.
Replaced the hardcoded "task name" on the modal with actual names of projects.
Communiti-Skill-Exchange 1
Communiti-Skill-Exchange 2
Communiti-Skill-Exchange 3

### CODE REVIEW

File: src/styles/assets/icons/MVP mascot.svg
Comment (chukwuemekasmba) [inline src/styles/assets/icons/MVP mascot.svg:1]: can this be put in the public/icons folder instead?

**Review by chukwuemekasmba — CHANGES_REQUESTED** [review]
Love the animations, but would recommend moving the mascot to the public folder before merging.

File: src/styles/assets/icons/MVP mascot.svg
Comment (Bita-Fotovvat) [reply src/styles/assets/icons/MVP mascot.svg:1]: Done.

**Comment by Bita-Fotovvat** [comment]
The requested change is done.

**Review by chukwuemekasmba — APPROVED** [review]

**Pull request merged**